client/signup/steps/rewind-add/index.jsx

```diff
@@ -0,0 +1,77 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import StepWrapper from 'signup/step-wrapper';
+import Card from 'components/card';
+import Button from 'components/button';
+import SignupActions from 'lib/signup/actions';
+import QueryRewindState from 'components/data/query-rewind-state';
+
+class RewindAddStep extends Component {
+	static propTypes = {
+		flowName: PropTypes.string,
+		goToNextStep: PropTypes.func.isRequired,
+		positionInFlow: PropTypes.number,
+		signupProgress: PropTypes.array,
+		stepName: PropTypes.string,
+	};
+
+	addCredentials = () => {
+		SignupActions.submitSignupStep( {
+			processingMessage: this.props.translate( 'Setting up your site' ),
+			stepName: this.props.stepName,
+		} );
+
+		this.props.goToStep( 'rewind-form' );
+	};
+
+	skipStep = () => this.props.goToNextStep();
+
+	renderStepContent = () => {
+		const { translate } = this.props;
+
+		return (
+			<Card className="rewind-add__card">
+				<QueryRewindState siteId={ this.props.siteId } />
+				<h3 className="rewind-add__title">{ translate( 'Add credentials' ) }</h3>
+				<img className="rewind-add__image" src="/calypso/images/illustrations/security.svg" />
+				<p className="rewind-add__description">
+					{ translate(
+						'By adding your site credentials, you are giving WordPress.com access to perform' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'By adding your site credentials, you are giving WordPress.com access to perform automatic actions on your server including backing up your site, restoring your site, as well as manually accessing your site in case of an emergency.' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).